Fixing JAX interface for adjoint diff (#1349)
PennyLaneAI/pennylane: .github/CHANGELOG.md

<h3>Bug fixes</h3>

> * Copy of `JacobianTape` now duplicates the `jacobian_options` attribute.  The jax interface
>  now supports adjoint differentiation.  A warning is raised when jax support for float64 is not
>  enabled but the user requests jax and finite diff together.
>   [(#1349)](https://github.com/PennyLaneAI/pennylane/pull/1349)

Reviewer: ```suggestion
* Copying the `JacobianTape` now correctly also copies the `jacobian_options` attribute. This fixes a bug
  allowing the JAX interface to support adjoint differentiation.  Secondly, a warning is raised when JAX
  support for `float64` is not enabled when using the JAX interface with `diff_method=finite_diff`.
  [(#1349)](https://github.com/PennyLaneAI/pennylane/pull/1349)
```

<h3>Bug fixes</h3>

* Copying the `JacobianTape` now correctly also copies the `jacobian_options` attribute. This fixes a bug
  allowing the JAX interface to support adjoint differentiation.  Secondly, a warning is raised when JAX
  support for `float64` is not enabled when using the JAX interface with `diff_method=finite_diff`.
  [(#1349)](https://github.com/PennyLaneAI/pennylane/pull/1349)

* Fixes drawing QNodes that contain multiple measurements on a single wire.
  [(#1353)](https://github.com/PennyLaneAI/pennylane/pull/1353)

> 

Author: ```suggestion
```

* Fixes drawing QNodes with no operations.
  [(#1354)](https://github.com/PennyLaneAI/pennylane/pull/1354)

> 

Author: ```suggestion
```
